docsets/PyTorch-Geometric/docset.json

```diff
@@ -21,5 +21,9 @@
                 "link": "https://github.com/seanexplode"
             }
         },
+        {
+            "version": "2.2.0",
+            "archive": "versions/2.2.0/PyTorch-Geometric.tgz"
```

Comment:
This archive doesn't exist. Please note archive paths are case-sensitive.

Also please add newer versions at the top of this list.